dali/ft/filecopy.cpp

```diff
@@ -2687,15 +2683,19 @@ class CApiCopyClient : public CInterfaceOf<IAPICopyClient>
             dbgassertex(storagePlane->getStorageType()==StorageType::StorageTypeAzureBlob);
             uri.append(".blob");
         }
-        const char *container = storagePlane->queryStorageContainer();
-        uri.appendf(".core.windows.net/%s", container);
+        uri.appendf(".core.windows.net/");
     }
-    static void buildAzureStorageURI(IDistributedFile *distFile, unsigned partNum, const char * baseURI, const char *sasToken, StringBuffer & uri)
+    static void buildAzureStorageURI(IDistributedFile *distFile, IStoragePlane * storagePlane, unsigned partNum, const char * baseURI, const char *sasToken, StringBuffer & uri)
     {
+        IDistributedFilePart & distFilePart = distFile->queryPart(partNum);
         StringBuffer path;
-        distFile->queryPart(partNum).getStorageFilePath(path, 0);
+        distFilePart.getStorageFilePath(path, 0);
```

Comment:
minor:  Clearer to have a local variable for copy to avoid asking the question why is 0 being passed to this function - 

unsigned copy = 0; // no platform replication
distFilePart.getStorageFilePath(path, copy);
...
unsigned stripeNum = distFilePart.getStripeNum(copy);

Reply:
should buildAzureStorageURI perhaps take a copyNum too, even if always 0 for now..